WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -46,9 +46,54 @@
           + "featured_image,"       // 17
           + "featured_video,"       // 18
           + "post_avatar,"          // 19
-          + "sort_index,"           // 20
-          + "date,"                 // 21
-          + "published,"            // 22
+          + "score,"                // 20
```

Comment:
Are we planning to use `score`? It seems it's not actually being used except for trivially reading/writing it to the DB. May I suggest removing it if we're not planning to use it in a follow up PR? Thanks!